wger/core/management/commands/add-user-rest-api.py

```diff
@@ -0,0 +1,35 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.contrib.auth.models import User
+from wger.core.models import ApiUser
+
+
+class Command(BaseCommand):
+    help = 'Command to create a new user via API'
+
+    def add_arguments(self, parser):
+        # add argurments/parameters to be used when creating a user
+        parser.add_argument('username', type=str)
+        parser.add_argument('email', type=str)
+        parser.add_argument('password', type=str)
+        parser.add_argument('created_by_username', type=str)
+
+    def handle(self, *args, **options):
+        # confirm created_by_username exists
+        created_by = User.objects.filter(username=options["created_by_username"])
+        if not created_by:
+            raise CommandError(' {} is not registered user.'.format(options["created_by_username"]))
+
+        if User.objects.filter(username=options["username"]):
+            raise CommandError('Username {} already exists.'.format(options["username"]))
+
+        if User.objects.filter(email=options["email"]):
+            raise CommandError("Email {} already exists.".format(options["email"]))
+
+        # if creator exists, username and email are not taken
+        new_user = User.objects.create_user(username=options["username"],
+                                            email=options["email"],
+                                            password=options["password"])
```

Comment:
from from the documentation the user model hashes the password.
when i check the database , the passwords are hashed